client.lua

```diff
@@ -0,0 +1,50 @@
+local QBCore = exports['qb-core']:GetCoreObject()
+local hasPhone = false
+
+local function DoPhoneCheck(PlayerItems)
+    hasPhone = false
```

Comment:
This pattern has a code smell, I can feel it in the force, beware when refactoring in the future.

Reply:
Yeah I agree, not the greatest way.

Reply:
Suggestions?

Reply:
> This pattern has a code smell, I can feel it in the force, beware when refactoring in the future.

If you have no suggestions to improve the smell, please cover your nose sir